### PR TITLE
[shoppingshadow.com] Re-enable ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

**Comment by youdly** [comment]
Dangling reference error pending https://github.com/EFForg/https-everywhere/pull/13773

File: src/chrome/content/rules/Shoppingshadow.com.xml [hunk 1]
```diff
@@ -1,18 +1,21 @@
 <!--
-	For other eBay coverage, see EBay.xml.
-
-
-	Fully covered domains:
+	img and di* have certificate mismatch issue, need to redirect to secure-*
```
Comment (J0WI) [inline src/chrome/content/rules/Shoppingshadow.com.xml]: Please also add test urls/rules for the `secure-` variants.

Why did you removed the reference to `EBay.xml`?

**Comment by youdly** [comment]
Added test url and rebased.

File: src/chrome/content/rules/Shoppingshadow.com.xml [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
-		- img.shoppingshadow.com	(→ img.shopping.com)
+	<rule from="^http://img(2)?\.shoppingshadow\.com/"
+		to="https://secure-img$1.shoppingshadow.com/" />
 
--->
-<ruleset name="shoppingshadow.com" default_off="mismatched">
+		<test url="http://img.shoppingshadow.com/" />
+		<test url="http://img2.shoppingshadow.com/" />
+		<test url="http://secure-img2.shoppingshadow.com/" />
+		<test url="http://secure-img2.shoppingshadow.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Shoppingshadow.com.xml]: Duped test url

**Comment by youdly** [comment]
Fix duplicate test. Rebased.

**Pull request merged**